### ISSUE TITLE
SeparatorWrap: add support for method reference operator

### ISSUE THREAD

**Issue by rnveach** [checkstyle/checkstyle#3951]
"::" is separator, NOT a operator - https://docs.oracle.com/javase/specs/jls/se8/html/jls-3.html#jls-3.11

$ cat MethodRef.java

``` java
import java.util.Objects;
import java.util.stream.Stream;

public class MethodRef
{
    {
        Stream.of().filter(Objects
            ::nonNull); // VIOLATION -- want '::' operator at end of previous line
    }
}
```

$ cat config.xml

``` xml
<!DOCTYPE module PUBLIC
  "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
  "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">

<module name="Checker">
    <module name="TreeWalker">
        <module name="SeparatorWrap">
            <message key="line.previous" value="Separtor ''{0}'' should be at end of previous line" />
            <property name="tokens" value="METHOD_REF" />
            <property name="option" value="eol" />
        </module>
    </module>
</module>
```

/var/tmp $ java -jar checkstyle-7.0-all.jar -c config.xml MethodRef.java
C:\Users\U0076777>java -jar checkstyle-7.0-all.jar -c config.xml MethodRef.java
com.puppycrawl.tools.checkstyle.api.CheckstyleException: cannot initialize module TreeWalker - Token "METHOD_REF" was
not found in Acceptable tokens list in check com.puppycrawl.tools.checkstyle.checks.whitespace.SeparatorWrapCheck
        at com.puppycrawl.tools.checkstyle.Checker.setupChild(Checker.java:407)
        at com.puppycrawl.tools.checkstyle.api.AutomaticBean.configure(AutomaticBean.java:138)
        at com.puppycrawl.tools.checkstyle.Main.runCheckstyle(Main.java:385)
        at com.puppycrawl.tools.checkstyle.Main.runCli(Main.java:323)
        at com.puppycrawl.tools.checkstyle.Main.main(Main.java:161)
Caused by: com.puppycrawl.tools.checkstyle.api.CheckstyleException: Token "METHOD_REF" was not found in Acceptable tokens list in check com.puppycrawl.tools.checkstyle.checks.whitespace.SeparatorWrapCheck
        at com.puppycrawl.tools.checkstyle.TreeWalker.registerCheck(TreeWalker.java:226)
        at com.puppycrawl.tools.checkstyle.TreeWalker.setupChild(TreeWalker.java:163)
        at com.puppycrawl.tools.checkstyle.api.AutomaticBean.configure(AutomaticBean.java:138)
        at com.puppycrawl.tools.checkstyle.Checker.setupChild(Checker.java:402)
        ... 4 more
Checkstyle ends with 1 errors.

---

need METHOD_REF added to OperatorWrap check to be able to enforce the wrapping policy.
##

**Comment by Pedro12909** [comment]
I am on it.

**Comment by Luolc** [comment]
I am on it.

**Comment by rnveach** [comment]
fix is merged
